src/test/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExampleTest.java

```diff
@@ -314,24 +253,14 @@ private void verifyChart(final String url,
     // Wait until we have 5 items available in the chart
```

Comment:
This comment is unnecessary (and now it's wrong)

Reply:
ack :)